### PR TITLE
feature/upgrade-caddy

### PR DESCRIPTION
**Pull request by jrouzierinverse**
# Description
Upgrade to caddy v2.6.1

# Issue
fixes #ISSUE_NUMBER

# Delete branch after merge
YES

# NEWS file entries
## Enhancements
* Upgrade to caddy v2.6.1

### CODE REVIEW

File: conf/caddy-services/httpdispatcher.conf.example [hunk 1]
```diff
@@ -1,43 +1,46 @@
...
 
+:8889 {
+	handle_path /common/* {
+		root * /usr/local/pf/html/common
+		file_server
+	}
 
-:8889/content {
-  root /usr/local/pf/html/captive-portal/content
-  mime {
```
Comment (julsemaan) [inline conf/caddy-services/httpdispatcher.conf.example:25]: It looks like this configuration wasn't migrated into this file

**Comment by julsemaan** [comment]
I see that this upgrades to go 1.19 in the go.mod file. Is this intended?

If so, what's the status about the pfpki being broken with go 1.19

**Comment by julsemaan** [comment]
Pipeline launched to check the results of the tests:
https://gitlab.com/inverse-inc/packetfence/-/pipelines/653343355

**Comment by fdurand** [comment]
> I see that this upgrades to go 1.19 in the go.mod file. Is this intended?
> 
> If so, what's the status about the pfpki being broken with go 1.19

https://go-review.googlesource.com/c/go/+/383215

**Comment by jrouzierinverse** [comment]
They reverted that change of rejecting duplicate extensions.

https://github.com/golang/go/issues/57556

Do we go with v1.19 or v1.20 for the next release?

**Comment by fdurand** [comment]
Can you rebase ?

**Pull request closed without merging**